src/main/java/org/olf/folio/order/util/MarcUtils.java

```diff
@@ -103,6 +103,8 @@ public String getVendorItemId(DataField nineSixtyOne ) {
         String vendorItemId= new String();
         if (nineSixtyOne != null) {
             vendorItemId = nineSixtyOne.getSubfieldsAsString(VENDORID);
+        } else {
+            return null;
```

Comment:
Just for my own benefit, shouldn't `vendorItemId` be `null` when initialized on line 103?

Reply:
I am considering revisiting all of the MarcUtils methods and how they're tested.  In this case, the method is returning null if the 961 field or the $i subfield is missing.  It could also return an empty string if the 961 field and $i subfield exists, but contains an empty string.  Although some methods return a default value (e.g. "olin" for getLocation